fs/rc/rcserver/rcserver.go

```diff
@@ -48,6 +48,7 @@ type Server struct {
 }
 
 func newServer(opt *rc.Options, mux *http.ServeMux) *Server {
+	opt.HTTPOptions.WebUI = true
```

Comment:
Doesn't this want to be conditional on opt.WebUI so something like

```go
opt.HTTPOptions.WebUI = opt.WebUI
```

Reply:
Yes. I'll correct that.